src/test/groovy/org/prebid/server/functional/util/PBSUtils.groovy

```diff
@@ -97,4 +96,9 @@ class PBSUtils implements ObjectMapperWrapper {
     static BigDecimal getRandomPrice(int min = 0, int max = 10, int scale = 3) {
         getRandomDecimal(min, max).setScale(scale, HALF_UP)
     }
+
+    static int getRandomNumberWithExcept(int exceptValue, int min = 0, int max = MAX_VALUE) {
```

Comment:
You can overload the existing `getRandomNumber()` method. Juss add new parameter to the end.

Reply:
We can't just overload method and add new parameter to the end since in `getRandomNumber()` two non mandatory parameters. My point of view it's better to have this one method with right naming.